tests/nlu/utilities.py

```diff
@@ -11,6 +11,15 @@ def write_file_config(file_config):
         return f
 
 
+def verify_sequence_not_none(sequence, sentence):
```

Comment:
Can we rename this to `get_feature_vectors` or `get_vectors_from_sequence_and_sentence_features`?

Reply:
Also, can you please add type annotations to this helper function?